src/app/infrastucture/image-source/image-source.module.ts

```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { ImageSourcePipe } from './image-source.pipe';
+
+@NgModule({
+  declarations: [ImageSourcePipe],
+  exports: [ImageSourcePipe],
+  imports: [CommonModule],
```

Comment:
It seems, you don't need common module here. If you don't use any functionality from CommonModule, then you shouldn't import it.